src/CabanaPD_Particles.hpp

```diff
@@ -445,8 +448,12 @@ class Particles<MemorySpace, PMB, Dimension>
         log( std::cout, "No particle output enabled." );
 #endif
 #endif
+
+        _timer.stop();
     }
 
+    auto time() { return _timer.time(); };
```

Comment:
@streeve We do not have here "auto timeInit() { return 0.0; };"

Reply:
Updated